### PR TITLE
Rename `ReplicationGroupDescription` to `Description`

### PR DESCRIPTION
**Pull request by RedbackThomson**
Issue #, if available: https://github.com/aws-controllers-k8s/community/issues/1250

Description of changes:
Renamed the `CreateReplicationGroup` and `ModifyReplicationGroup` input field `ReplicationGroupDescription` to simply be `Description`, to match the `DescribeReplicationGroups` output shape

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/e2e/scenarios/ScaleUpAndDown/cme_scale_down_rollback.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ steps:
       spec:
         engine: redis
         replicationGroupID: scaling$RANDOM_SUFFIX
-        replicationGroupDescription: Scalind down rollback
+        description: Scaling down rollback
```
Comment (jaypipes) [inline test/e2e/scenarios/ScaleUpAndDown/cme_scale_down_rollback.yaml:18]: heh, nice spelling catch :)

**Review by jaypipes — APPROVED** [review]
:+1:

**Comment by jaypipes** [comment]
@elops interesting. the ReplicationGroup's CRD still lists the field as `Description`:

https://github.com/aws-controllers-k8s/elasticache-controller/blob/c3182807fd72b55c7d8954e092a5fb6bf45bdba0/config/crd/bases/elasticache.services.k8s.aws_replicationgroups.yaml#L162-L164

and that is what is listed in the required fields:

https://github.com/aws-controllers-k8s/elasticache-controller/blob/c3182807fd72b55c7d8954e092a5fb6bf45bdba0/config/crd/bases/elasticache.services.k8s.aws_replicationgroups.yaml#L401

@RedbackThomson any idea (from a Helm perspective) that this might have changed?

**Comment by elops** [comment]
Ok I am a bit wiser now, so I installed chart 0.0.25 first. This one installed CRDs so helm upgrade (actually downgrade) to 0.0.24 didn't do much. Only when I uninstalled ack, deleted elasticache related crds manually, installed v0.0.24 could I use description again.

**Comment by elops** [comment]
Sorry guys, I figured it out. I installed actually first version 0.0.9 which was super old and CRDs from that one prevailed until I deleted them manually and installed more recent version of controller.

**Comment by jaypipes** [comment]
No worries @elops, it happens! :) Glad to hear you figured it out!

**Pull request merged**